modules/mobile/docs/schemas/Locations.yml

```diff
@@ -27,4 +30,5 @@ properties:
             example: "COLUMBUS VAMC"
           address:
```

Comment:
AFAICT this doesn't cause problems, but just as an FYI, in many places we essentially duplicate information. For example, here we have `type: object` and the additional properties, then we have those again within the FacilityAddress file. I'm pretty sure redocly is smart enough to de-dupe it.

Reply:
good catch. ill remove those in my next PR with more specs